inference-engine/src/legacy_api/include/convert_function_to_cnn_network.hpp

```diff
@@ -17,5 +17,10 @@ namespace details {
 INFERENCE_ENGINE_API_CPP(std::shared_ptr<CNNNetworkImpl>)
 convertFunctionToICNNNetwork(const std::shared_ptr<const ::ngraph::Function>& graph, const ICNNNetwork &network);
 
+INFERENCE_ENGINE_API_CPP(std::shared_ptr<CNNNetworkImpl>)
+convertFunctionToICNNNetwork(const std::shared_ptr<const ::ngraph::Function>& graph,
+                             const ICNNNetwork &network, bool keep_constant_inputs);
+
```

Comment:
can we extend existing function with default parameter?

Reply:
If I remember right Vladimir (who started these changes) faced with some issues with default parameter and friend member and it is an adaptation of those changes. Maybe I am wrong.

Reply:
https://stackoverflow.com/questions/23333949/the-correct-way-to-define-default-argument-for-a-friend-function-in-c

To use friend for function with default argument, you need to put the function declaration with default argument before the class and use full arguments list without default values in friend declaration.

Reply:
Done.